client/reader/components/quick-post/index.tsx

```diff
@@ -35,14 +36,17 @@ interface PostItem {
 	site_ID: number;
 	title: string;
 	content: string;
+	URL: string;
 }
 
 function QuickPost( {
 	primarySiteId,
 	receivePosts,
+	successNotice,
 }: {
 	primarySiteId: number | null;
 	receivePosts: ( posts: PostItem[] ) => Promise< void >;
+	successNotice: ( message: string, options: object ) => void;
```

Comment:
1. Instead of using Redux can we please use it via `dispatch( successNotice( args ) );`
2. Seems like we can do the same for `primarySiteId` and `receivePosts` as well.

Reply:
I think either way it is still redux, just the method of connection.

Currently these all use the connect HOC that was common for class based components, since we are functional now we have the opportunity to just use the useDispatch hook inside the component itself instead. I wouldn't hard block the PR on it, but its quick cleanup to consider while here.

Reply:
I'll handle this refactor separately 👍